dsp-meta/src/api/router.rs

```diff
@@ -18,33 +19,32 @@ use crate::app_state::AppState;
 /// Having a function that produces our router makes it easy to call it from tests
 /// without having to create an HTTP server.
 pub fn router(shared_state: Arc<AppState>) -> Router {
+    let cors = CorsLayer::new()
+        .expose_headers(Any)
+        .allow_methods(Any)
+        .allow_headers(Any)
+        .allow_origin(Any);
     Router::new()
         .route(
-            "/api/projects",
+            "/api/v1/projects",
```

Comment:
praise: great! I'm in favour of adding versioning to all public APIs

Reply:
Thanks, but no praises for me please. This was the API in the go service, I have simply aligned the code to be compatible with the existing frontend.